jetbrains-ultimate/it/software/aws/toolkits/jetbrains/services/lambda/go/GoLocalRunConfigurationIntegrationTest.kt

```diff
@@ -170,11 +181,9 @@ class GoLocalRunConfigurationIntegrationTest(private val runtime: LambdaRuntime)
 
     @Test
     fun samIsExecutedWithDebugger() {
-        // TODO enable when go debugging is fixed on sam cli public release
-        // see: https://github.com/aws/aws-sam-cli/issues/2462
-        assumeTrue(false)
+        // only run this test on > 2020.1
+        assumeFalse(ApplicationInfo.getInstance().let { info -> (info.majorVersion == "2020" && info.minorVersionMainPart == "1") })
```

Comment:
Add our min version marker to remove this on 201 dropping

Reply:
Fixed